### ISSUE TITLE
release-react publish server error

### ISSUE THREAD

**Issue by bguedel** [electrode-io/electrode-ota-server#5]
```
electrode-ota-server/server/routes/apps.js:86
                        params:{app, deployment}, auth:{credentials:{email}},
                                                                     ^

TypeError: Cannot match against 'undefined' or 'null'.
```

When running through the [setup guide](http://www.electrode.io/docs/electrode_react_native_over_the_air_electron.html) when running the [publish](http://www.electrode.io/docs/electrode_react_native_over_the_air_electron.html#publishing-react) step I am getting the above error.

The error is actually on [this](https://github.com/electrode-io/electrode-ota-server/blob/master/server/routes/apps.js#L84) line; such that ```server.app.config.app``` is undef. This breaks the server and the intended default is not used.

For a temp fix I did the following,
```js
const {
    params:{app, deployment}, auth:{credentials:{email}},
    //server:{app:{config:{app:{downloadUrl = request.server.info.uri + '/storagev2/'}}}},
    payload
} = request;
const downloadUrl = request.server.info.uri + '/storagev2/';
```

Is this the intended flow? Was there a different config step that I missed?

**Comment by jspears** [comment]
Let me double check.   I suspect that the app config does not have it configured.

**Comment by jspears** [comment]
Looks like I should update the docs to have a line like this, in the config.  
I updated the default.js so it has an app so this does not fail and published a new version 1.0.3
If you don't want to update add a line like this to your config.

```
  "app": {
    "electrode": true
  }
```

**Comment by bguedel** [comment]
that did it! thanks
